### PR TITLE
build-info: update Gluon to 2023-12-26

### PR DESCRIPTION
**Pull request by blocktrron**
Update Gluon from c2dc338a to db0e7bda.

```
db0e7bda Merge pull request #3054 from blocktrron/flexible-feature-packages
683defd0 gluon-tls: add package (#3114)
43aff415 docs: fix spacing inconsistency
f2b2f81a ath79-generic: add support for Sophos AP55c (#3119)
3ab7149f Merge pull request #3061 from T-X/pr-pi4-support
c286964f ath79-generic: add support for Sophos AP100c (#3117)
74681fcd mesh-vpn-core: remove old migrations (#3113)
dbea6f97 bcm27xx-bcm2711: add (broken) support for the Raspberry Pi 4
7e5a2350 bcm27xx: switch from ext4 to squashfs
dd5c5033 image-customization: load image-customization.lua once
83406bc5 build image-customization: create init routine
9a8ee573 Merge pull request #3111 from blocktrron/upstream-master-updates
8956b9d7 modules: update gluon
b360669e modules: update packages
065dedae modules: update openwrt
20c2adae bcm27xx: update manifest aliases (#3110)
0f044103 docs: fix spelling and referencing
1c45dd21 image-customization: include file-extension
280b4375 image-customization: simplify return values
3152ce6a build: inform users of deprecation
64234f75 docs: rewrite docs to represent image-customization
edca7850 image-customization: allow negation of selections
83923003 build: only evaluate list of site-selections once
f99fa1ba docs: add documentation for image-customization
3d9e60c7 build: allow modification of device build-properties
3d07353c features: update comment
d1ef488b target-lib: remove unused function
f629b71a build: implement flexible site-selection system
```

Signed-off-by: GitHub Actions <info@darmstadt.freifunk.net>

Supersedes #13

### CODE REVIEW

**Review by herbetom — CHANGES_REQUESTED** [review]
`ffda-network-setup-mode` for the NWA55AXE is missing

File: image-customization.lua [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	'respondd',
+	'status-page',
+	'web-advanced',
+	'web-logging',
+	'web-private-wifi',
+	'web-wizard',
+})
+
+packages({
+	'ca-bundle',
```
Comment (herbetom) [inline image-customization.lua]:
```suggestion
```
`ca-bundle` is part of the tls feature https://github.com/freifunk-gluon/gluon/commit/683defd0f8597773f1f3f1fb2031cb010c82ab56

Is there a reason to include it regardless on all devices, even tiny?

File: image-customization.lua [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+	'iwinfo',
+	'libustream-mbedtls',
```
Comment (herbetom) [inline image-customization.lua]:
```suggestion
```

same

**Review by herbetom — APPROVED** [review]

**Comment by herbetom** [comment]
We've previously had `usbutils` included. Is that something that's already a dependency of something else or do we simply don't "need" it anymore?

We've also had `pkgs_usb_net` on many more targets. But that's probably fine for now to ignore and only to add it if there is an actual need.

**Comment by blocktrron** [comment]
usbutils contains lsusb. The USB-Net packages make no sense on regular router targets, additionally they are not supportable with the new network abstraction in Gluon.

**Pull request merged**